geoopt/manifolds/base.py

```diff
@@ -521,6 +521,16 @@ def _retr_transp(self, x, u, t, v, *more):
             out = out + (self._transp_one(x, u, t, v),)
         return out
 
+    def rand(self, shape, size):
```

Comment:
It is a public method, should have a complete documentation

Reply:
1. size can be ommited as inferred from shape
2. arguments passed should be like `rand(*sizes)` to match pytorch api

Reply:
3. should check shape via `self._check_shape method`